### PR TITLE
Support -> Support Us

### LINKED ISSUE #386: Change "Support" in the dropdown menu to "Support Us"

### PR DESCRIPTION
**Pull request by FatherInire**
Fixes #386

### CODE REVIEW

File: files/templates/header.html [hunk 1]
```diff
@@ -118,7 +118,7 @@
 
 								<a class="dropdown-item" rel="nofollow noopener noreferrer" href="https://github.com/themotte/rDrama"><i class="fab fa-github fa-fw mr-3"></i>Source Code</a>
 
-								<a class="dropdown-item" href="/support"><i class="fas fa-hippo fa-fw mr-3"></i>Support {{SITE_NAME}}</a>
```
Comment (zorbathut) [inline files/templates/header.html:121]: Oh man, I apologize for changing the requirements here, but now that I see it *should* have been SITE_NAME, y'know what, that's a better option. But then why isn't that working? On my local codebase it just shows up as blank.

image

Any way you can track down why SITE_NAME isn't working instead? It should say "Support The Motte", and I'd rather not hardcode that if possible, but this *should* have worked and isn't.

(I probably just did something dumb here honestly)

**Review by zorbathut — CHANGES_REQUESTED** [review]

**Comment by zorbathut** [comment]
I can't seem to actually add this to the review, but please remove those .pyc files! They shouldn't be checked in, I think that's because of attempting to run it locally :)

File: files/templates/header.html (hunk 1 above)
Comment (FatherInire) [reply files/templates/header.html:121]: looks like it should be SITE_TITLE not SITE_NAME - fixed

**Comment by FatherInire** [comment]
>I can't seem to actually add this to the review, but please remove those .pyc files! They shouldn't be checked in, I think that's because of attempting to run it locally :)

Oops. I've added *.pyc to .gitignore in the other PR

**Review by zorbathut — APPROVED** [review]

**Pull request merged**